vector/src/main/java/im/vector/util/SystemUtils.kt

```diff
@@ -122,6 +122,20 @@ fun startNotificationSettingsIntent(fragment: Fragment, requestCode: Int) {
         val uri = Uri.fromParts("package", fragment.activity?.packageName, null)
         intent.data = uri
     }
-
     fragment.startActivityForResult(intent, requestCode)
 }
+
+fun supportNotificationChannels() = (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O)
```

Comment:
I prefer this method to be in NotificationUtils.kt, because it does not really interfere with the system

Reply:
moved back